nucliadb_sdk/nucliadb_sdk/tests/test_ask.py

```diff
@@ -142,3 +145,23 @@ def test_ask_response_parser():
     assert ask_response.metadata.tokens.output == 5
     assert ask_response.metadata.timings.generative_first_chunk == 0.1
     assert ask_response.metadata.timings.generative_total == 0.2
+
+
+def test_ask_synchronous(docs_dataset, sdk: nucliadb_sdk.NucliaDB):
+    sdk.session.headers["X-Synchronous"] = "true"
+    resp = sdk.ask(
+        kbid=docs_dataset,
+        query="Nuclia loves Semantic Search",
+    )
+    assert isinstance(resp, SyncAskResponse)
+    sdk.session.headers.pop("X-Synchronous", None)
+
+
+def test_ask_stream(docs_dataset, sdk: nucliadb_sdk.NucliaDB):
+    sdk.session.headers["X-Synchronous"] = "false"
+    resp = sdk.ask(
+        kbid=docs_dataset,
+        query="Nuclia loves Semantic Search",
+    )
+    assert isinstance(resp, SyncAskResponse)
```

Comment:
Do we always return a `SyncAskResponse`?

Reply:
Yup, this is the idea. In the same way as chat